From e818c4fb4d3bac6aaf997dbb85c854688f6c239a Mon Sep 17 00:00:00 2001
From: Stanislaw Gruszka <sgruszka@redhat.com>
Date: Wed, 10 Dec 2014 10:29:27 -0500
Subject: [wireless] iwlwifi: Add missing PCI IDs for the 7260 series

Message-id: <1418207389-7899-14-git-send-email-sgruszka@redhat.com>
Patchwork-id: 101587
O-Subject: [RHEL7.1 PATCH 13/35] iwlwifi: Add missing PCI IDs for the 7260 series
Bugzilla: 1155538
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: John Greene <jogreene@redhat.com>

From: Oren Givon <oren.givon@intel.com>

BZ#1155538

commit 4f08970f5284dce486f0e2290834aefb2a262189 upstream.

Add 4 missing PCI IDs for the 7260 series.

Signed-off-by: Oren Givon <oren.givon@intel.com>
Signed-off-by: Emmanuel Grumbach <emmanuel.grumbach@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Brew:
https://brewweb.devel.redhat.com/taskinfo?taskID=8330268

Testing:
Compiled only.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/wireless/iwlwifi/pcie/drv.c b/drivers/net/wireless/iwlwifi/pcie/drv.c
index 98950e4..78eaa48 100644
--- a/drivers/net/wireless/iwlwifi/pcie/drv.c
+++ b/drivers/net/wireless/iwlwifi/pcie/drv.c
@@ -273,6 +273,8 @@ static DEFINE_PCI_DEVICE_TABLE(iwl_hw_card_ids) = {
  {IWL_PCI_DEVICE(0x08B1, 0x4070, iwl7260_2ac_cfg)},
  {IWL_PCI_DEVICE(0x08B1, 0x4072, iwl7260_2ac_cfg)},
  {IWL_PCI_DEVICE(0x08B1, 0x4170, iwl7260_2ac_cfg)},
+ {IWL_PCI_DEVICE(0x08B1, 0x4C60, iwl7260_2ac_cfg)},
+ {IWL_PCI_DEVICE(0x08B1, 0x4C70, iwl7260_2ac_cfg)},
  {IWL_PCI_DEVICE(0x08B1, 0x4060, iwl7260_2n_cfg)},
  {IWL_PCI_DEVICE(0x08B1, 0x406A, iwl7260_2n_cfg)},
  {IWL_PCI_DEVICE(0x08B1, 0x4160, iwl7260_2n_cfg)},
@@ -316,6 +318,8 @@ static DEFINE_PCI_DEVICE_TABLE(iwl_hw_card_ids) = {
  {IWL_PCI_DEVICE(0x08B1, 0xC770, iwl7260_2ac_cfg)},
  {IWL_PCI_DEVICE(0x08B1, 0xC760, iwl7260_2n_cfg)},
  {IWL_PCI_DEVICE(0x08B2, 0xC270, iwl7260_2ac_cfg)},
+ {IWL_PCI_DEVICE(0x08B1, 0xCC70, iwl7260_2ac_cfg)},
+ {IWL_PCI_DEVICE(0x08B1, 0xCC60, iwl7260_2ac_cfg)},
  {IWL_PCI_DEVICE(0x08B2, 0xC272, iwl7260_2ac_cfg)},
  {IWL_PCI_DEVICE(0x08B2, 0xC260, iwl7260_2n_cfg)},
  {IWL_PCI_DEVICE(0x08B2, 0xC26A, iwl7260_n_cfg)},
-- 
1.7.1